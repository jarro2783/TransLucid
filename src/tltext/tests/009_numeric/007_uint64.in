%%
uint64"0";;
uint64"1";;
uint64"12";;
uint64"18446744073709551615";;

uint64"0" + uint64"0";;
uint64"15" + uint64"0";;
uint64"0" + uint64"15";;
uint64"8" + uint64"5";;

uint64"0" - uint64"0";;
uint64"10" - uint64"0";;
uint64"15" - uint64"3";;

uint64"6" * uint64"1";;
uint64"6" * uint64"0";;

uint64"5" / uint64"1";;
uint64"23" / uint64"5";;

uint64"7" % uint64"4";;

uint64"0" < uint64"1";;
uint64"1" < uint64"1";;
uint64"2" < uint64"1";;

uint64"0" <= uint64"1";;
uint64"1" <= uint64"1";;
uint64"2" <= uint64"1";;

uint64"0" > uint64"1";;
uint64"1" > uint64"1";;
uint64"2" > uint64"1";;

uint64"0" >= uint64"1";;
uint64"1" >= uint64"1";;
uint64"2" >= uint64"1";;

uint64"0" == uint64"1";;
uint64"1" == uint64"1";;
uint64"2" == uint64"1";;

uint64"0" != uint64"1";;
uint64"1" != uint64"1";;
uint64"2" != uint64"1";;

uint64"5"!prec;;
uint64"5"!is_signed;;
