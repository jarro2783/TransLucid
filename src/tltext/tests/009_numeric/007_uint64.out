uint64"0"
uint64"1"
uint64"12"
uint64"18446744073709551615"
uint64"0"
uint64"15"
uint64"15"
uint64"13"
uint64"0"
uint64"10"
uint64"12"
uint64"6"
uint64"0"
uint64"5"
uint64"4"
uint64"3"
true
false
false
true
true
false
false
false
true
false
true
true
false
true
false
true
false
true
64
false
